python/ray/tune/suggest/bayesopt.py

```diff
@@ -4,13 +4,16 @@
 
 import copy
 
-try:
-    import bayes_opt as byo
-except Exception:
-    byo = None
-
 from ray.tune.suggest.suggestion import SuggestionAlgorithm
 
+byo = None
+
+
+def _import_bayesopt():
```

Comment:
What is this function used for?

Reply:
it seems like a lot of these import statements were time sinks (~1 sec. to run), so we decided to put them in functions and only call them when needed.

Reply:
good catch though, I forgot to call it..